spec/features/users/06_user_edits_account_info_spec.rb

```diff
@@ -0,0 +1,100 @@
+require "rails_helper"
+
+# Acceptance Criteria:
+# [x] user can see current account details page
+# [x] user can navigate to edit account page
+# [x] user can update name
+# [x] user can update email
+# [x] user can update profile picture
+# [x] user can update password
+# [x] user must enter current password for update to persist
+
+feature "edit account details" do
+
+  let (:user) { FactoryGirl.create(:user) }
+  let! (:lifehack) { FactoryGirl.create(:lifehack, creator: user) }
+  let! (:review) do
+    FactoryGirl.create(:review, creator: user, lifehack: lifehack)
```

Comment:
Are your factories creating their dependent objects? Could cut down on the instantiated objects here if you wanted by just creating `review` and then calling its associated `lifehack` later.